client/state/reader/follows/reducer.js

```diff
@@ -177,7 +181,12 @@ export const items = createReducer(
 			// Only check items with an ID (the subscription ID) because those are what
 			// we show on the manage listing. Items without an ID are either inflight follows
 			// or follows that we picked up from a feed, site, or post object.
-			return omitBy( state, follow => follow.ID && ! seenSubscriptions.has( follow.feed_URL ) );
+			return omitBy(
+				state,
+				( follow, key ) =>
+					key !== prepareComparableUrl( follow.feed_URL ) ||
```

Comment:
maybe an earlier screw up in how we unfollowed things? or how we moved from one feed url to a new one?